spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -39,6 +39,8 @@
     )
 
     expect(response.valid?(original_challenge, original_origin)).to eq(true)
+    expect(response.attestation_type).to eq(WebAuthn::AttestationStatement::Base::ATTESTATION_TYPE_BASIC_OR_ATTCA)
```

Comment:
I only see "Basic" as the returned type in the "Verification procedure" in https://www.w3.org/TR/2018/CR-webauthn-20180807/#fido-u2f-attestation. I may be missing something.